eox_core/api/v1/serializers.py

```diff
@@ -189,9 +190,10 @@ def validate(self, attrs):
         """
         # If at least one of these conditions is true, then the user can't be updated.
         safe_fields = getattr(settings, "EOX_CORE_USER_UPDATE_SAFE_FIELDS", [])
+        unsafe_meta_fields = getattr(settings, 'EOX_CORE_USER_UPDATE_UNSAFE_META_FIELDS', [])
 
         for attr in attrs:
-            if attr not in safe_fields:
+            if attr not in safe_fields or attr in unsafe_meta_fields:
```

Comment:
I don't understand why we need this if we have `EOX_CORE_USER_UPDATE_SAFE_FIELDS = ['email']` this won't raise any exception but if we have `EOX_CORE_USER_UPDATE_SAFE_FIELDS = []` this will raise the exception so why do we need to do `EOX_CORE_USER_UPDATE_UNSAFE_META_FIELDS= ['email']` if the result is the same with `EOX_CORE_USER_UPDATE_SAFE_FIELDS = []`

Reply:
You are right, the setting is unnecessary